# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../bdb33/gcc43.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur db-3.3.11-orig/include/db_cxx.h db-3.3.11/include/db_cxx.h
--- db-3.3.11-orig/include/db_cxx.h	2010-07-06 16:34:03.000000000 +0200
+++ db-3.3.11/include/db_cxx.h	2010-07-06 16:46:05.000000000 +0200
@@ -49,12 +49,14 @@
 // Forward declarations
 //
 
-#include <iostream.h>
 #include <stdarg.h>
 #include "db.h"
 #include "cxx_common.h"
 #include "cxx_except.h"
 
+#include <iostream>
+using namespace std;
+
 class Db;                                        // forward
 class Dbc;                                       // forward
 class DbEnv;                                     // forward
